src/tests/ftest/ior/ior_intercept_messages.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2019-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import re
+import os
+from ior_test_base import IorTestBase
+
+
+class IorInterceptMessages(IorTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test class Description: Runs IOR with interception.
+
+       Look for messages provided by the library
+
+    :avocado: recursive
+    """
+
+    def test_ior_intercept_messages(self):
+        """Jira ID: DAOS-7647.
+
+        Test Description:
+            Purpose of this test is to run ior using dfuse with interception
+            library enabled and look for some debug messages provided
+            by the library.
+
+        Use case:
+            Run ior with dfuse flags set on debug
+            Look for interception library messages.
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,small
+        :avocado: tags=daosio,dfuse,il
+        :avocado: tags=iorinterceptmessages
+        """
+        d_il_report_value = self.params.get("value",
+                                            "/run/tests/D_IL_REPORT/*")
+        intercept = os.path.join(self.prefix, 'lib64', 'libioil.so')
+        summary_pattern = "^\[libioil\] Performed [0-9]+ reads and [0-9]+ " \
```

Comment:
(lint) '\['. String constant might be missing an r prefix.
(lint) '\]'. String constant might be missing an r prefix.

Reply:
You can avoid this by adding 'r' to the pattern string:
`r"^\[libioil\] Performed [0-9]+ reads and [0-9]+ writes from [0-9]+ files*"`

Reply:
Fixed